src/api/mock.rs

```diff
@@ -556,6 +556,7 @@ mod test {
     };
 
     #[test]
+    #[cfg(not(feature = "color"))]
```

Comment:
When running tests with cargo hack, this test failed because the output was colorized, instead of being in the expected format.